web/src/components/MapView.tsx

```diff
@@ -43,7 +43,7 @@ const OverlayContainer = styled.div`
   display: flex;
   padding: 12px 8px;
   flex: 1;
-  z-index: 1;
+  z-index: 0;
```

Comment:
As the default is zero, this could be omitted, correct?
```suggestion
```

Reply:
Good point!